lively.collab/comments/commentMorph.js

```diff
@@ -217,11 +217,38 @@ export class CommentMorph extends Morph {
 
     this.isInEditMode = false;
 
+    this.resetButtons();
+    this.ui.editSaveButton.tooltip = 'Edit Comment';
```

Comment:
Maybe put this into resetButtons? (resolveButton tooltip is also set there)

Reply:
Nvm, got tricked by the name `resetButtons`. It'd be better to leave this tooltip out of the method